http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -828,9 +854,11 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
         _ <- deleteDisk
```

Comment:
Should be able to delete all three above in parallel?

Reply:
I'm not sure (the TODO was from me). It's probably safer to wait for the `helm uninstall` to finish before deleting the underlying nodepool/disk.